### DISCUSSION TITLE
Questions about the use of the MIT license: Do we need to share the profit with the original author?

### DISCUSSION THREAD

**Discussion by vocal29** [community/community#29256] [New to GitHub] [answered]
Hi,
We inherited the MIT license and code of repository A, developed a product in repository B, and made commercial profits. 

Under the MIT license, do we need to return part of the profits to the original owner of repository A?

**Comment by airtower-luna** [comment, marked as answer]
[Read the license](https://choosealicense.com/licenses/mit/), it's very short and clear. And if anything's unclear I recommend you consult a lawyer.
